python/ray/train/data_parallel_trainer.py

```diff
@@ -338,7 +338,7 @@ def training_loop(self) -> None:
             name=session.get_trial_name(),
             id=session.get_trial_id(),
             resources=session.get_trial_resources(),
-            logdir=os.getcwd(),
+            logdir=session.get_trial_dir(),
```

Comment:
This is still a needed change in this PR, since setting `chdir_to_trial_dir=False` in the Tuner would make `os.getcwd()` return the original working directory here, which is not the trial logdir.